src/api/rpc_api/state.rs

```diff
@@ -59,6 +59,13 @@ pub trait GetStorage {
 		storage_item: &'static str,
 	) -> Result<StorageKey>;
 
+	fn get_storage_double_map_key_prefix<K: Encode>(
```

Comment:
Could you add an explanatory comment? Something like 
`/// Retrieve the key prefix for a storage double map with the first key included.`